lib/internal/fs/promises.js

```diff
@@ -290,49 +286,34 @@ class FileHandle extends EventEmitter {
       throw new ERR_INVALID_STATE('The FileHandle is locked');
     this[kLocked] = true;
 
```

Comment:
```suggestion
    if (options.type !== undefined) {
      validateString(options.type, 'options.type');
      if (options.type !== 'bytes') {
        process.emitWarning('...', 'ExperimentalWarning');
      }
    }
```

Reply:
Okay, I only just now realized that this `type` was only ever passed to the `ReadableStream` constructor if it was `'bytes'`. So it was in fact allowed to pass a non-`'bytes'` `type` to this function; the fact that the `ReadableStream` constructor [throws with other values does not matter](https://github.com/nodejs/node/pull/55461#discussion_r1831922443). I can add the warning in this case, yes.